tap/src/main.rs

```diff
@@ -59,7 +59,7 @@ fn main() {
     ethernet::capture::print_devices();
     
     let metrics = Arc::new(Mutex::new(metrics::Metrics::new()));
-    let bus = Arc::new(Bus::new(metrics.clone(), "ethernet_packets".to_string()));
+    let bus = Arc::new(Bus::new(metrics.clone(), "ethernet_packets".to_string(), configuration.general.wifi_pkt_buffer_capacity, configuration.general.ethernet_pkt_buffer_capacity));
```

Comment:
Should we pass the entire `configuration` here? I have a feeling we'll use more config variables in the `Bus` in the future.

Reply:
Sounds good. will change and add the commits.

Reply:
done.